test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -31,20 +31,22 @@
 
 (deftest ^:integration legacy-routes
   (testing "The legacy web routing service properly handles old routes."
-    (bootstrap/with-puppetserver-running app
-      {}
-      (is (= 200 (:status (http-get "/v2.0/environments"))))
-      (is (= 200 (:status (http-get "/production/node/localhost"))))
-      (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     app
+     {}
+     (is (= 200 (:status (http-get "/v2.0/environments"))))
+     (is (= 200 (:status (http-get "/production/node/localhost"))))
+     (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
```

Comment:
For the `environments` and `node` endpoints, yeah, that would be true.  If you're concerned enough about it, I'll convert it back to using a real JRuby - and add some more assertions around the body of the response as well.